index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+//const readFiles = require('./lib/read-files.js');
```

Comment:
Good job attempting the bonus, but the code you pushed up was only using your ```readManyFiles``` function instead of your ```readFiles``` function, so I got an error because you're utilizing the ```async``` Node module, but you didn't install it as a dependency. 

Great job doing the research to find a library that will help you handle async, but remember to read the docs carefully and see if you need to ```npm i -S <foo>``` so that the dependency gets required. In this case, what you'd do is...

```npm i -S async``` inside your terminal.

Then whatever module you use ```async``` in, you would require it in like so...

```const async = require('async');```